src/libostree/ostree-sysroot.h

```diff
@@ -268,4 +268,9 @@ gboolean ostree_sysroot_simple_write_deployment (OstreeSysroot *sysroot, const c
                                                  OstreeSysrootSimpleWriteDeploymentFlags flags,
                                                  GCancellable *cancellable, GError **error);
 
+_OSTREE_PUBLIC
+gboolean
+ostree_sysroot_deployment_kexec_load (OstreeSysroot *self, OstreeDeployment *deployment,
```

Comment:
When adding new API you'll need to also add this `libostree-devel.sym`. There's a few subtleties around this and it's probably easier if I just add a commit for you to apply to your branch, will do in a bit.